.stickler.yml

```diff
@@ -0,0 +1,5 @@
+linters:
+  rubocop:
+    fixer: true
+fixers:
+ enable: true
```

Comment:
Needs one extra space here to align properly with the rest of the file, I think you are using 2 spaces for intendation, not 1 [`OPTIONAL`]